app/matchers/bulkrax/application_matcher_decorator.rb

```diff
@@ -1,21 +1,25 @@
 # frozen_string_literal: true
 # OVERRIDE Bulkrax 4.4.2 Application Parser here to make sure atla custom parse methods get added
-Bulkrax::ApplicationMatcher.class_eval do
-  def process_parse
-    # OVERRIDE here to make sure atla custom parse methods get added
-    # New parse methods will need to be added here
-    parsed_fields = ['remote_files',
-                     'language', 'subject', 'types', 'model', 'resource_type',
-                     'format_original', 'format_digital', 'date']
-    # This accounts for prefixed matchers
-    parser = parsed_fields.find { |field| to&.include? field }
-    if @result.is_a?(Array) && self.parsed && self.respond_to?("parse_#{parser}")
-      @result.each_with_index do |res, index|
-        @result[index] = send("parse_#{parser}", res.strip)
+module Bulkrax
+  module ApplicationMatcherDecorator
+    def process_parse
+      # OVERRIDE here to make sure atla custom parse methods get added
+      # New parse methods will need to be added here
+      parsed_fields = ['remote_files',
+        'language', 'subject', 'types', 'model', 'resource_type',
+        'format_original', 'format_digital', 'date']
+      # This accounts for prefixed matchers
+      parser = parsed_fields.find { |field| to&.include? field }
+      if @result.is_a?(Array) && self.parsed && self.respond_to?("parse_#{parser}")
+        @result.each_with_index do |res, index|
+          @result[index] = send("parse_#{parser}", res.strip)
+        end
+        @result.delete(nil)
+      elsif self.parsed && self.respond_to?("parse_#{parser}")
+        @result = send("parse_#{parser}", @result)
       end
-      @result.delete(nil)
-    elsif self.parsed && self.respond_to?("parse_#{parser}")
-      @result = send("parse_#{parser}", @result)
     end
   end
 end
+
+Bulkrax::ApplicationMatcher.prepend(Bulkrax::ApplicationMatcherDecorator)
```

Comment:
I'm unsure why this works now when it didn't last week....but a win is a win! 🤷🏾‍♀️

Reply:
also don't know why because this is exactly how we had this file!
I remember it wasn't even registering that this decorator existed